plugins/nginx-vhosts/install

```diff
@@ -52,6 +52,13 @@ chgrp -R adm /var/log/nginx
 gpasswd -M "" dokku
 [[ -f /etc/logrotate.d/nginx ]] && sed -i -e 's/create 0640 www-data dokku/create 0640 www-data adm/g' /etc/logrotate.d/nginx
 
+# Create nginx error templates
+sudo mkdir -p /etc/nginx/dokku-errors
+sudo cp ./templates/400-error.html /etc/nginx/dokku-errors/400-error.html
```

Comment:
Also, instead of `/etc/nginx/dokku-errors`, maybe instead `$DOKKU_LIB_ROOT/data/nginx-vhosts/dokku-errors`?